security/security/src/main/java/io/helidon/security/DefaultAuditProvider.java

```diff
@@ -30,13 +30,33 @@
  */
 final class DefaultAuditProvider implements AuditProvider {
     private final Logger auditLogger;
+    private final Level failureLevel;
+    private final Level successLevel;
+    private final Level infoLevel;
+    private final Level warnLevel;
+    private final Level errorLevel;
+    private final Level auditFailureLevel;
+
+    private DefaultAuditProvider(Config config) {
+        // config node is already located on the security node
+        this.auditLogger = Logger.getLogger(config.get("audit.defaultProvider.logger").asString().orElse("AUDIT"));
+        this.failureLevel = level(config, "failure", Level.FINEST);
+        this.successLevel = level(config, "success", Level.FINEST);
+        this.infoLevel = level(config, "info", Level.FINEST);
+        this.warnLevel = level(config, "warn", Level.WARNING);
+        this.errorLevel = level(config, "error", Level.SEVERE);
+        this.auditFailureLevel = level(config, "audit-failure", Level.SEVERE);
+    }
 
-    private DefaultAuditProvider(String loggerName) {
-        this.auditLogger = Logger.getLogger(loggerName);
+    private Level level(Config config, String auditSeverity, Level defaultLevel) {
+        return config.get("audit.defaultProvider.level.failure")
```

Comment:
Should this be `"audit.defaultProvider.level." + auditSeverity` ?

Reply:
Fixed